### PR TITLE
PL.Editor #162 Filter empty bold tags and instruct

### LINKED ISSUE #162: Filter out or prevent empty bold tags in WYSIWYG (woofmark)
Often when bolding things in the WYSIWYG module, empty bold markdown gets saved, resulting in sections like this which no longer even render as Markdown:

```
An initial paragraph

**
**

**A real bolded section here**

Continue with another paragraph
```

We should try to catch and filter out this:

```
**
**
```

perhaps within the module (as future WYSIWYG alternative modules may not share this bug). This should be pretty easy to detect and filter!

### PR DESCRIPTION
**Pull request by rexagod**
Fixes #162 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by rexagod** [comment]
@publiclab/reviewers

**Comment by jywarren** [comment]
Oh wow. Would a test of this be possible, do you think? Like, inserting some blank bold tags in a test and confirming that they (but not used bold tags like `**this phrase**`) are removed? Thank you so much!!!

If you're interested, a test could make for a good first-timers-only issue, to build a community here as @gauravano has suggested!

**Comment by rexagod** [comment]
Hey that's a great idea! I'll draft one up.

**Comment by rexagod** [comment]
Sure.

File: src/modules/PublicLab.RichTextModule.js [hunk 1]
```diff
@@ -168,6 +168,19 @@ module.exports = PublicLab.RichTextModule = PublicLab.Module.extend({
...
 
+        crossvent.add(_module.wysiwyg.textarea, 'keyup', function (e) {
+      _editor.validate();
+      var regexp= /\*\*[\n]+\*\*/g;
+      if (_module.wysiwyg.mode == "markdown" && _module.value().match(regexp) && $('.invalid-bold-tags-warning').length === 0) {
+         var message = "Invalid input: Please remove all invalid bold tags like the ones below:<br><br>**<br>**";
+        _module.wysiwyg.textarea.innerHTML = (_module.wysiwyg.textarea.innerHTML).replace(regexp,'');
+        $(_module.wysiwyg.textarea).after("<div id='scrollpointBold' class='invalid-bold-tags-warning alert alert-warning'>" + message + "</div>");
+        $('html, body').animate({
+        scrollTop: $("#scrollpointBold").offset().top
```
Comment (rexagod) [inline src/modules/PublicLab.RichTextModule.js]: @jywarren Should I make the same changes here that you mentioned in #200?

**Comment by rexagod** [comment]
Should I add the timestamps fix to this one too first? Or is it okay?

**Comment by jywarren** [comment]
Yes, we'd best, thanks!

**Comment by jywarren** [comment]
Awesome. Link to this from the first-timers issue and we'll have the test reference this PR, which could help future debugging! Thanks!!!

**Comment by rexagod** [comment]
@jywarren @gauravano @SidharthBansal Is #202 eligible for a `fto`? Please suggest any changes that I should implement in order to make it one. Thanks!

**Pull request merged**